### DISCUSSION TITLE
Visual Studio 2022 Uno Platform project template support

### DISCUSSION THREAD

**Discussion by jindal1979** [unoplatform/uno#15206] [General]
Is the Visual Studio 2022 Uno Platform project template getting updates? I saw the link  https://marketplace.visualstudio.com/items?itemName=unoplatform.uno-platform-addin-2022 and the last update was in November last year.

**Comment by jeromelaban** [comment]
Thanks for the question. It is getting updates as needed, generally when we release a new minor or major version of Uno, but most of the contents is either automatically updated through nuget templates, or through other nuget packages,

Did you notice an issue?

**Reply by jindal1979** [reply]
No, I didn't encounter any issues. 

Earlier, the Visual Studio 2022 template was getting updates as soon as the new version of Uno.Templates were available.  Now, the CLI template is at version 5.0.39 however Visual Studio 2022 is at version 5.0.17.30.
